src/components/Patient/ManagePatients.tsx

```diff
@@ -966,11 +966,11 @@ export const PatientManager = () => {
                 navigate(`facility/${selectedFacility.id}/discharged-patients`);
                 break;
               default:
-                console.error("Unknown dialog state");
+                Notification.Error({ msg: "Unknown dialog state" });
```

Comment:
You missed one of the earlier comments (make sure to resolve each one, helps with keeping track as well).

No need for a default state here 🤔 

Also you don't need to request review and tag - one of those is enough 😅

Reply:
Ok Sir, Can we connect in LinkedIn ? It will be a wonderful learning curve for me to work with you sir